Circles/Views/UIA/UsernameEnrollForm.swift

```diff
@@ -12,61 +12,83 @@ import Matrix
 struct UsernameEnrollForm: View {
     var session: SignupSession
     @State var username: String = ""
-    
     @State var showAlert = false
     @State var alertTitle = ""
     @State var alertMessage = ""
+    @State private var showHint = false
     
     enum FocusField {
         case username
     }
     @FocusState var focus: FocusField?
     
     var body: some View {
-        VStack(alignment: .center, spacing: 20) {
-            Spacer()
-
-            Text("Choose a username")
-                .font(.title2)
-                .fontWeight(.bold)
-
-            TextField("Username", text: $username, prompt: Text("username"))
+        let screenWidthWithOffsets = UIScreen.main.bounds.width - 72
+        BasicImage(name: SystemImages.launchLogoPurple.rawValue)
+            .frame(width: 125, height: 43)
+            .padding(.bottom, 30)
+        
+        VStack(alignment: .leading) {
+            Text("Choose your User ID")
+                .font(
+                    CustomFonts.nunito14
+                        .weight(.bold)
+                )
+                .foregroundColor(Color.greyCool1100)
+            
+            TextField("@username:us.domain.com", text: $username)
```

Comment:
Since the user doesn't really get to pick the full user id string (the domain part is beyond their control) let's change the prompt here to simply "username".